designing-reports/report-designer-tools/desktop-designers/visual-studio-report-designer/vs-designer-structure.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Structure
+page_title: Visual Studio Report Designer Structure
+description: "Learn about the structure of the Visual Studio Report Designer, and how to use its main features when editing report definitions."
+slug: visual-studio-report-designer-structure
+tags: visual,studio,report,designer,tool,structure
+published: True
+position: 5
+---
+
+# Structure of the Visual Studio Report Designer
+
+After you have opened a report in the Telerik Visual Studio Report Designer, you may use the following elements it exposes:
+
+## Telerik Reporting Menu
+
+The [menu]({%slug telerikreporting/designing-reports/report-designer-tools/desktop-designers/visual-studio-report-designer/telerik-reporting-menu%}) lets you run the [Report Explorer]({%slug telerikreporting/designing-reports/report-designer-tools/desktop-designers/tools/report-explorer%}), [Data Explorer]({%slug telerikreporting/designing-reports/report-designer-tools/desktop-designers/tools/data-explorer%}), [Group Explorer]({%slug telerikreporting/designing-reports/report-designer-tools/desktop-designers/tools/group-explorer%}), [Report Wizard]({%slug telerikreporting/designing-reports/report-designer-tools/desktop-designers/tools/report-wizards/band-report-wizard/overview%}) and [Upgrade Wizard]({%slug telerikreporting/designing-reports/report-designer-tools/desktop-designers/visual-studio-report-designer/upgrade-wizard%}). It is accessible through the __Telerik Menu --> Reporting__ for Visual Studio versions up to 2017. For Visual Studio 2019 and later through the __Extensions Menu --> Telerik --> Reporting__.
+
+## Design Views Buttons
+
+Use these buttons to switch between __Design__, __Preview__, and __HTML__ view mode of the report.
+
+## Report Selector Button
+
+Located in the upper left-hand of the report designer. Clicking this button makes the report active in the __Properties__ window.
+
+## Rulers
+
+They are on the top and left side of the designer and provide a point of reference to the report layout.
+
```

Comment:
Maybe add images for those buttons? Or one section for those with one images and be listed with bullets?